staging/src/k8s.io/client-go/pkg/apis/clientauthentication/v1beta1/types.go

```diff
@@ -46,6 +46,9 @@ type ExecCredentialSpec struct {
 	// ExecConfig.ProvideClusterInfo).
 	// +optional
 	Cluster *Cluster `json:"cluster,omitempty"`
+
+	// Interactive declares whether stdin has been passed to this exec plugin.
+	Interactive bool `json:"interactive"`
```

Comment:
Open question: do we like this contract with the exec plugin?

Some discussion here: https://kubernetes.slack.com/archives/C0EN96KUY/p1613768159114200?thread_ts=1613767745.112500&cid=C0EN96KUY.